From c4def85e03e234e8c6c7c58ccdfb35faa7191a63 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ji=C5=99=C3=AD=20Kone=C4=8Dn=C3=BD?=
 <DragonLichcz@gmail.com>
Date: Tue, 12 Jan 2016 19:32:58 +0100
Subject: [PATCH 1/2] Add appdata.xml

Added appdata.xml which is used for software center.

http://people.freedesktop.org/~hughsient/appdata/
---
 etc/PlayOnLinux.appdata.xml | 39 +++++++++++++++++++++++++++++++++++++++
 1 file changed, 39 insertions(+)
 create mode 100644 etc/PlayOnLinux.appdata.xml

diff --git a/etc/PlayOnLinux.appdata.xml b/etc/PlayOnLinux.appdata.xml
new file mode 100644
index 0000000..5f6d53b
--- /dev/null
+++ b/etc/PlayOnLinux.appdata.xml
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Jiri Konecny <dragonlichcz@gmail.com> -->
+<component type="desktop">
+  <id>PlayOnLinux.desktop</id>
+  <project_license>GPL-3.0</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>PlayOnLinux</name>
+  <summary>Graphical front-end for Wine</summary>
+  <description>
+    <p>
+    PlayOnLinux will allow you to run your favorite Windows games and
+    applications on Linux through Wine easily.
+    </p>
+    <p>
+    New users can often find Wine to be intimidating and difficult to use.
+    PlayOnLinux simplifies much of this and makes installing and using
+    Windows programs easier.
+    </p>
+    <p>
+    PlayOnLinux have the database of Windows applications from which user
+    can install desired application with few clicks. It will automatically
+    setup your Wine prefix and download required redistributable libraries
+    if needed.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://jkonecny.fedorapeople.org/images/playonlinux_screen_01.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://jkonecny.fedorapeople.org/images/playonlinux_screen_02.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://jkonecny.fedorapeople.org/images/playonlinux_screen_03.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://www.playonlinux.com/</url>
+  <update_contact>dragonlichcz_at_gmail.com</update_contact>
+</component>
-- 
2.5.0

